tutorials/setting-up-loraserver.md

```diff
@@ -0,0 +1,634 @@
+INFO[0000] gateway/gcp_pub_sub: setting up client
+INFO[0000] gateway/gcp_pub_sub: setup downlink topic     topic=eu868-gateway-commands
+INFO[0001] gateway/gcp_pub_sub: setup uplink topic       topic=eu868-gateway-events
+INFO[0002] gateway/gcp_pub_sub: check if uplink subscription exists  subscription=eu868-gateway-events-loraserver
+INFO[0002] gateway/gcp_pub_sub: create uplink subscription  subscription=eu868-gateway-events-loraserver
+INFO[0005] applying database migrations
+INFO[0006] migrations applied                            count=19
+INFO[0006] starting api server                           bind="0.0.0.0:8000" ca-cert= tls-cert= tls-key=
+```
+If all is well, then you can start the service in the background using:
+
+```bash
+sudo systemctl start loraserver
+sudo systemctl enable loraserver
+```
+
+### Install LoRa App Server
+
+When you have completed all previous steps, then it is time to install the
+last component, [LoRa App Server](https://www.loraserver.io/lora-app-server/). This is the
+application-server that provides a web-interface for device management and
+will publish application-data to a Pub/Sub topic.
+
+#### Create Pub/Sub topic
+
+In the GCP console, navigate to **Pub/Sub > Topics**. Then click on the
+**Create topic** button to create a topic named `lora-app-server`.
+
+#### Install LoRa App Server
+
+SSH to the VM and execute the following command to install LoRa App Server:
+
+```bash
+sudo apt install lora-app-server
+```
+
+#### Configure LoRa App Server
+
+The LoRa App Server configuration file is located at
+`/etc/lora-app-server/lora-app-server.toml`. Below you will find a minimal
+but working configuration example. Please refer to the LoRa App Server
+[Configuration](/lora-app-server/install/config/) documentation for all the
+available options.
+
+You need to replace the following values:
+
+* **[PASSWORD]** with the `loraserver_as` PostgreSQL user password
+* **[POSTGRESQL_IP]** with the **Primary IP address** of the created PostgreSQL instance
+* **[REDIS_IP]** with the **IP address** of the created Redis instance
+* **[JWT_SECRET]** with your own random JWT secret (e.g. the output of `openssl rand -base64 32`)
+
+##### Configuration example
+
+```toml
+[postgresql]
+dsn="postgres://loraserver_as:[PASSWORD]@[POSTGRESQL_IP]/loraserver_as?sslmode=disable"
+
+[redis]
+url="redis://[REDIS_IP]:6379"
+
+[application_server]
+
+  [application_server.integration]
+  backend="gcp_pub_sub"
+
+  [application_server.integration.gcp_pub_sub]
+  project_id="lora-server-tutorial"
+  topic_name="lora-app-server"
+
+  [application_server.external_api]
+  bind="0.0.0.0:8080"
+  tls_cert="/etc/lora-app-server/certs/http.pem"
+  tls_key="/etc/lora-app-server/certs/http-key.pem"
+  jwt_secret="[JWT_SECRET]"
+```
+
+To test if there are no errors, you can execute the following command:
+
+```bash
+sudo lora-app-server
+```
+
+This should output something like (it is important that there are no errors):
```

Comment:
The parenthetical phrase here seems misplaced. I'm not sure what revision to suggest, since I am not completely certain what you are trying to communicate here.

Here is my best attempt:
'This should output something like (it is important that there are no errors):'
->
'This should output something like the following; it is important that there are no errors:'

Reply:
Actually it is already mentioned in a sentence above that the user should keep an eye on errors. I'm removing it.